examples/nextjs-cldimage/app/app-demo/_components/cld-image.jsx

```diff
@@ -0,0 +1,8 @@
+'use client';
+
+import { CldImage as CldImageDefault } from 'next-cloudinary';
+
+/** @param {import('next-cloudinary').CldImageProps} props */
```

Comment:
curious about using this syntax, is there a reason this is preferable than importing CldImageProps and typing it on the `props: CldImageProps`?

Reply:
oh - this is a JS file - thats right, nevermind!